### DISCUSSION TITLE
How do I include a system package installed by apt install?

### DISCUSSION THREAD

**Discussion by bryanhughes** [python-poetry/poetry#9287] [Q&A]
Hello,

I am working on a Nvidia Jetson Orin that has its own versions of several libraries like `tensorrt`. They are installed to `/usr/lib/python3.10/dist-packages/tensorrt`

```
$ ll /usr/lib/python3.10/dist-packages/tensorrt
total 3656
-rw-r--r-- 1 root root    5302 Sep 25  2023 __init__.py
-rw-r--r-- 1 root root 3732064 Sep 25  2023 tensorrt.so
```

I tried the following in my pyproject.toml:

```
packages = [
    { include = "/usr/lib/python3.10/dist-packages/tensorrt" }
]```

which does nothing

also tried 
```
[tool.poetry.dependencies]
tensorrt = {path = "/usr/lib/python3.10/dist-packages/tensorrt"}
```
Which does gives me:
```
Directory /usr/lib/python3.10/dist-packages/tensorrt for tensorrt does not seem to be a Python package
```

Any help would be greatly appreciated.

Cheers,
Bryan

**Comment by dimbleby** [comment]
https://python-poetry.org/docs/configuration#virtualenvsoptionssystem-site-packages

but be aware that if after locking poetry disagrees with the answer that you have provided, then it will try to put it right - which probably is not what you want.  you may prefer to `poetry lock` but `pip install`

**Comment by bryanhughes** [comment]
Thanks!
